sites/all/themes/dccr2015/templates/views/views-view-unformatted--sponsors.tpl.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * View that shows the sponsors grouped by Level
+ *
+ * @ingroup views_templates
+ */
+?>
+<?php if (!empty($title)): ?>
+  <h3 class="<?php print(strtolower($title)); ?>"><?php print $title; ?></h3>
+<?php endif; ?>
+
+<div class="sponsors-wrapper">
+	<?php foreach ($rows as $id => $row): ?>
+	  <div<?php if ($classes_array[$id]) { print ' class="' . $classes_array[$id] .'"';  } ?>>
```

Comment:
This is a template based on the views unformatted template, it is provided by the views UI "Template information" area. The only change I made from the template was to add the group class to the <H3> element to have the ability to style the group headline (place the sponsorship level icon according to the designs). All the other stuff is the same as the template provided by the Views Module.